### PR TITLE
Minor update to MAB

### PR DESCRIPTION
**Pull request by izawwlgood**
Revised to skill: Utility and added expiration messaging.

### CODE REVIEW

File: data/base-spells.yaml [hunk 1]
```diff
@@ -1129,9 +1129,10 @@ spell_data:
     mana: 15
     recast: 1
   Magnetic Ballista:
-    skill: Targeted Magic
+    skill: Utility
```
Comment (rpherbig) [inline data/base-spells.yaml]: ❓ Isn't MAB both TM and Utility? If so, why the change?

Comment (BinuDR) [reply data/base-spells.yaml]: Probably so that it will recast on the expire message.
